File: Code Coach/Safety Deposit Boxes/SafetyDepositBoxes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] items = Console.ReadLine().Split(",");
            string item = Console.ReadLine();
            Console.WriteLine((Array.IndexOf(items, item) + 1) * 5);
        }
    }
}

File: Code Coach/Safety Deposit Boxes/SafetyDepositBoxes.java
import java.util.Scanner;
import java.util.Arrays;

public class SafetyDepositBoxes {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String[] items = input.nextLine().split(",");
        String item = input.nextLine();
        System.out.println((Arrays.asList(items).indexOf(item) + 1) * 5);
        input.close();
    }
} 